plonky2x/core/src/frontend/uint/uint32_n.rs

```diff
@@ -139,12 +139,12 @@ macro_rules! make_uint32_n {
                 let self_targets = self
                     .limbs
                     .iter()
-                    .map(|x| U32Target(x.0 .0))
+                    .map(|x| U32Target(x.variable.0))
                     .collect::<Vec<_>>();
                 let rhs_targets = rhs
                     .limbs
                     .iter()
-                    .map(|x| U32Target(x.0 .0))
+                    .map(|x| U32Target(x.variable.0))
```

Comment:
relatedly, i think we should make `U32Target` pub(crate) so that it's only accessible via U32Variables